packages/shift-next-routes/src/express/menu-handler.js

```diff
@@ -1,11 +1,25 @@
 const { SHIFTClient } = require('@shiftcommerce/shift-node-api')
+const MenuCache = require('../lib/menu-cache')
 const { setSurrogateHeaders } = require('../lib/set-cache-headers')
 
 module.exports = {
-  getMenu: async (req, res) => {
-    const response = await SHIFTClient.getMenusV1(req.query)
-
-    setSurrogateHeaders(response.headers, res)
```

Comment:
we still need to set the cache keys for the menu - doing this only for times we go to the API will prevent them from making it in